Gibe.DittoProcessors/Processors/NodesAttribute.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using System.Web.Mvc;
+using Examine;
+using Gibe.UmbracoWrappers;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class NodesAttribute : TestableDittoProcessorAttribute
```

Comment:
What is this one trying to achieve? I don't know why I'd stick [Nodes] on a property it's a meaningless name